sky/cli.py

```diff
@@ -1280,18 +1281,32 @@ def status(all: bool, refresh: bool):  # pylint: disable=redefined-builtin
     autostopped.
     """
     cluster_records = core.status(refresh=refresh)
-    filtered_cluster_records = []
-    reserved_clusters = []
+    nonreserved_cluster_records = []
+    reserved_clusters = collections.defaultdict(list)
```

Comment:
Is it possible to simplify this group based logic? Maybe specializing to just 1 spot controller?

Reply:
Sounds good. Changed each group to only has 1 cluster.